Gemfile

```diff
@@ -19,7 +19,10 @@ gem 'json-schema', '~> 2.8', '>= 2.8.1'
 gem 'json_api_client', '~> 1.16', '>= 1.16.1'
 gem 'pg', '>= 0.18', '< 2.0'
 gem 'puma', '~> 4.1'
+gem 'rack', '~> 2.1.2'
```

Comment:
Is this version needed, otherwise we should already have 2.1.1 as part of actionpack. see Gemfile.locl